webui/react/src/components/Page.tsx

```diff
@@ -33,16 +37,31 @@ const breadCrumbRender = (route: Route, params: unknown, routes: Route[]) => {
   );
 };
 
+const getFullSeoTitle = (title?: string, clusterName?: string) => {
+  return (title ? title+' - ':'')+
```

Comment:
this results in something like this correct? `title - : - clustername : determined`

thoughts on just separating with `-`? as a side we could also `Array.join` the parts on `-`

Reply:
Changed, you're right, that's much more readable. Thanks.